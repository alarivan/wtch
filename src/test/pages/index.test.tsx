import React from "react"
import { render, cleanup } from "@testing-library/react"
import IndexPage from "../../pages/index"

jest.mock("../../components/seo", () => {
  return () => "SEO Mock"
})

describe("IndexPage", () => {
  afterEach(cleanup)

  it("renders correctly", () => {
    const { asFragment } = render(<IndexPage />)

    expect(asFragment()).toMatchSnapshot()
  })
})
