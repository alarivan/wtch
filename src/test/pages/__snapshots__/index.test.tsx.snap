// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage renders correctly 1`] = `
<DocumentFragment>
  .c0 {
  background: rebeccapurple;
  margin-bottom: 1.45rem;
}

.c1 {
  margin: 0 auto;
  max-width: 960px;
  padding: 1.45rem 1.0875rem;
}

.c2 {
  margin: 0 auto;
}

.c3 {
  color: white;
  -webkit-text-decoration: none;
  text-decoration: none;
}

<header
    class="c0"
  >
    <div
      class="c1"
    >
      <h1
        class="c2"
      >
        <a
          class="c3"
          href="/"
        >
          Gatsby Basic TypeScript Starter
        </a>
      </h1>
    </div>
  </header>
  .c0 {
  margin: 0 auto;
  max-width: 960px;
  padding: 0 1.0875rem 1.45rem;
}

<div
    class="c0"
  >
    <main>
      SEO Mock
      <h1>
        Hi people
      </h1>
      <p>
        Welcome to your new Gatsby site.
      </p>
      <p>
        Now go build something great.
      </p>
    </main>
    <footer>
      Â© 2019, Built with 
      <a
        href="https://www.gatsbyjs.org"
      >
        Gatsby
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
